rosdep/base.yaml

```diff
@@ -3408,6 +3408,17 @@ libvtk-qt:
     xenial: [libvtk6-qt-dev]
     yakkety: [libvtk6-qt-dev]
     zesty: [libvtk6-qt-dev]
+libvulkan-dev:
+  debian:
```

Comment:
as this is available on all debian active distros, this can be simplified to: `debian: [libvulkan-dev]`

Reply:
Fixed in 56ff3d5.